### DISCUSSION TITLE
v2.224

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#370] [Announcements]
> [!WARNING]  
> During `dev` cycle some of patches were refactored. 
> If you face any issues, crashes, etc., clear patch options (AKA options.json) or data/cache in your patching tool. 

React in #308 to change defaults.

**Comment by 54m4d** [comment]
Is it possible to hide `Account` and `Your data` in the settings in this version?

**Reply by anddea** [reply]
No, not possible yet.

**Reply by 54m4d** [reply]
This is what confused me: `Settings: Update Insert position to work with "Account" and "Your data" and fallback to "About" if specified key does not exist`

**Reply by anddea** [reply]
It's about RVX preference. Now you can select where to put that setting, for example if you want to move ReVanced Extended setting from bottom to top, you can select General or Parent settings in options of your patching tool, and that way you can put it wherever you want. 

Edit: The first word means the patch name: "Settings: ... ". If it was hiding those settings elements it would be "Tuck away preferences: ..."

**Comment by Jman-Github** [comment]
Was the hook download button under the video player removed? I can't seem to find it in settings, and it brings up the default menu when I tap the download button.

**Reply by anddea** [reply]
Yes. It's in Extended > Overlay buttons > External download button (Enable), External downloader settings > Hook download button. You can disable External download button after that, the hook will stay.

**Reply by anddea** [reply]
If you want it to be enabled by default, you can add the idea here #308.

**Reply by Jman-Github** [reply]
Alright, but is there a way to hide that download button on the player, but have it work with the download button under the player?

**Reply by anddea** [reply]
Yes, just disable external download button. 

Edit: the hook will stay, hook is only related to original download button.

**Reply by Jman-Github** [reply]
Okay, thank you.

**Comment by 54m4d** [comment]
@anddea Hello Aaron, enabling change package name for Reddit makes it not possible to log in to the app, it says username or password is incorrect even if the credentials are correct, unfortunately I cannot provide any logs because there are no errors during patching, any tips?

**Reply by anddea** [reply]
Hi. What was the last working version? Or was there any version that worked before?

**Reply by 54m4d** [reply]
`2024.15.0` is the version that I had working, I skipped `2024.16.0` and the one I tested today was `2024.17.0` (Latest) with the login issue.

Could you please confirm if you're encountering the same issue on one of the two latest versions?

**Reply by anddea** [reply]
Yes, 2024.17 does not allow to log in. I'll take a look later.

**Reply by 54m4d** [reply]
The same goes for the older 2024.15. There must be an issue with the `change package name` patch.

**Reply by selfmusing** [reply]
Patch v2023.20.x & the latest reddit app

First install the older version, then login your account, now install the latest version.

This is the same login issue which even happens on official Snapchat and WhatsApp due to google signature verification checks .

**Reply by anddea** [reply]
2024.15 lets me to log in.

**Reply by 54m4d** [reply]
Can you try to change the package name to `anddea.reddit` and try again? it's strange if that works because it didn't on my device.

**Reply by anddea** [reply]
Yes, didn't work with clean install. 🤔

**Reply by 54m4d** [reply]
`change package name` is a universal patch that can be used with many other apps, I'm worried that apps won't work as expected if this patch is used, or maybe this issue is only related to Reddit, I hope it's the latter.

Yeah, seems like clean install won't let you log in, someone suggested `2023.20.0`, maybe this is the latest version that is compatible with `change package name`, I haven't tried.

**Reply by selfmusing** [reply]
Checkout this @54m4d  : https://youtu.be/4fJE_xmPshg?si=bc2PRFqJgrP3iI8P

**Reply by 54m4d** [reply]
@selfmusing Are there any patches that work with WhatsApp? Have you tried any?

**Comment by selfmusing** [comment]
> Imported your settings, no crash for me. I have latest patches, latest supported YT version.
> 
> Try other patching tool and also older version with latest patches and let me know if it's the same for you.


Tool used : RVX Manager
App version : v19.16.38
Patch version : v2.224.1-dev.1

Getting the same error in crash logs :
https://bin.kv2.dev/~662c31f95de7ed6bfdc23ff2

https://github.com/anddea/revanced-patches/assets/109387456/aa524899-ffe2-4cf1-9158-dc5ddc5eaced


EDIT: It also crashes, after changing custom seekbar color or If I only enable/disable the custom seekbar toggle ( With default settings of app after clean installation )

**Comment by hiru12** [comment]
May I know what to put in revanced manager to use your patches?
![Screenshot_20240504_045219_ReVanced Manager](https://github.com/anddea/revanced-patches/assets/74873448/9384107a-46bf-4385-adfa-d039e5de4d31)
![Screenshot_20240504_045215_ReVanced Manager](https://github.com/anddea/revanced-patches/assets/74873448/736ca9b0-798e-4701-8acd-30401f8519fc)

**Reply by selfmusing** [reply]
![Screenshot_20240504-131430_RVX Manager](https://github.com/anddea/revanced-patches/assets/109387456/68e7d084-b507-4138-81f9-9feb481f0257)
![Screenshot_20240504-131434_RVX Manager](https://github.com/anddea/revanced-patches/assets/109387456/4dd046bd-063e-4a91-8c7f-0957dd148663)
![Screenshot_20240504-131437_RVX Manager](https://github.com/anddea/revanced-patches/assets/109387456/73608c1b-8643-40f5-b365-513627088424)
![Screenshot_20240504-131441_RVX Manager](https://github.com/anddea/revanced-patches/assets/109387456/07de0ecf-c500-4664-9ec5-d7ab48246091)

**Reply by anddea** [reply]
Btw, those fields are case-insensitive.

Also, here's a pinned discussion for your question https://github.com/anddea/revanced-patches/discussions/215.

**Reply by 54m4d** [reply]
@selfmusing Which Manager is this?

**Reply by anddea** [reply]
Probably this one by Francesco146

https://github.com/Francesco146/revanced-manager/releases

**Reply by hiru12** [reply]
Thank you veru much! This is the best!

**Reply by 54m4d** [reply]
@anddea Btw, when I open the Manager and in the update section at the top, it almost always stuck "Loading" when I come back after some hours then it's not stuck anymore, I've always had this issue, and I'm using the latest one by inotia.

Do you also have this issue? Any idea why this happens?

**Reply by anddea** [reply]
> it almost always stuck "Loading" 

It's the same for me, never looked t it closely but maybe it happens because it gets rate limited. It can be fixed with an optimized code and local caching but I don't contribute to or maintain the Manager.

**Reply by 54m4d** [reply]
I see, it's a bit strange not sure how does fetching the updates makes GH rate limit it? inotia released the manager yesterday with all the fixes from the original revanced code, but it's still an issue.

**Comment by 54m4d** [comment]
@anddea Hello Aaron, quick question, Could you quickly confirm if any patches are compatible with WhatsApp? A while ago, I attempted to use the patch that `removes screenshot restriction`, but afterward, I was not able to log in to WA due that it detected something wrong with the app and threw an error.

**Reply by anddea** [reply]
This is more related to original ReVanced, as RVX doesn't support anything other than YouTube, YT Music and Reddit. 

All these apps are definitely checking for ways to improve their anti-modding policy, so anything can happen.

**Reply by 54m4d** [reply]
Thanks

**Comment by manojkumarcrazy** [comment]
> > [!WARNING]  
> > During `dev` cycle some of patches were refactored. 
> > If you face any issues, crashes, etc., clear patch options (AKA options.json) or data/cache in your patching tool. 
> 
> React in #308 to change defaults.

Video buffering issue how to solve please tell any solution

**Reply by anddea** [reply]
> Video buffering issue how to solve please tell any solution 

In Extended > Misc, you have 2 options for now, spoof stream data and test client, turn one of them, both have limitations, read respective description for each setting.
